pkgs/default.nix

```diff
@@ -288,4 +289,9 @@ in rec {
 
     NIX_CFLAGS_COMPILE = "-I${imagemagick}/include/ImageMagick-6";
   };
+
+  virtualbox = linuxPackages: (newScope linuxPackages) ./virtualbox {
```

Comment:
`nixpkgs` seems to enumerate all possible kernels which is something we want to avoid